app/controllers/schedules_controller.rb

```diff
@@ -4,7 +4,7 @@ class SchedulesController < ApplicationController
 
   def new
     @schedule = @guider.schedules.build(
-      start_at: 6.weeks.from_now
+      start_at: 6.weeks.from_now + 1.day
```

Comment:
Ok I couldn't do that without patching Relation. See the latest force push though. @benlovell